package.json

```diff
@@ -4,58 +4,57 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build systelab-translate --prod",
+    "build": "ng build systelab-translate --configuration production",
     "test": "ng test --watch=false --code-coverage --browsers=ChromeHeadless",
     "lint": "ng lint",
     "e2e": "ng e2e"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^12.1.1",
-    "@angular/common": "^12.1.1",
-    "@angular/compiler": "^12.1.1",
-    "@angular/core": "^12.1.1",
-    "@angular/forms": "^12.1.1",
-    "@angular/platform-browser": "^12.1.1",
-    "@angular/platform-browser-dynamic": "^12.1.1",
-    "@angular/router": "^12.1.1",
+    "@angular/animations": "^14.2.3",
+    "@angular/common": "^14.2.3",
+    "@angular/compiler": "^14.2.3",
+    "@angular/core": "^14.2.3",
+    "@angular/forms": "^14.2.3",
+    "@angular/platform-browser": "^14.2.3",
+    "@angular/platform-browser-dynamic": "^14.2.3",
+    "@angular/router": "^14.2.3",
     "@ngx-translate/core": "^13.0.0",
```

Comment:
There is a new ngx-translate 14.0 version

Reply:
Yes, I know, but this is not the scope of this update. These change will be done in the future.